ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts

```diff
@@ -12,11 +12,92 @@ const triggerSet: TriggerSet<Data> = {
   triggers: [],
   timelineReplace: [
     {
-      locale: 'en',
-      replaceText: {
+      'locale': 'en',
+      'replaceText': {
         'Sinbound Fire III/Sinbound Thunder III': 'Sinbound Fire/Thunder',
       },
     },
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Fatebreaker(?!\')': 'fusioniert(?:e|er|es|en) Ascian',
+        'Fatebreaker\'s Image': 'Abbild des fusionierten Ascians',
+      },
+      'replaceText': {
+        'Blastburn': 'Brandstoß',
+        'Blasting Zone': 'Erda-Detonation',
+        'Burn Mark': 'Brandmal',
+        'Burnished Glory': 'Leuchtende Aureole',
+        'Burnout': 'Brandentladung',
+        'Burnt Strike': 'Brandschlag',
+        'Cyclonic Break': 'Zyklon-Brecher',
+        'Explosion': 'Explosion',
+        'Fall of Faith': 'Sünden-Erdspaltung',
```

Comment:
```suggestion
        'Fall Of Faith': 'Sünden-Erdspaltung',
```

I'm so sorry, could I also get you to run the capitalization on "of" and "of the" for the FRU replacements as well?

Reply:
Should i also turn "Turn of the Heavens" into "Turn Of The Heavens" (in FRU)?

Reply:
Same with "Dawn of an Age" in Zoraal Ja

Reply:
Yes please.